dbt/include/trino/macros/materializations/materialized_view.sql

```diff
@@ -0,0 +1,80 @@
+{% materialization trino_materialized_view, adapter="trino" %} 
```

Comment:
Materialization can be called `materialized_view`, as we are already in the dbt-trino adapter.

Reply:
Done